test/extension/branches.test.ts

```diff
@@ -59,21 +59,21 @@ describe('Branches', () => {
             gitRoot: '',
             name: 'localBranch3',
             remote: '',
-            remoteType: 2,
+            remoteType: undefined,
```

Comment:
Local branches do not have any remotes, hence `remoteType=undefined`.
This is a change in behavior